pilot/pkg/xds/filters/filters.go

```diff
@@ -157,6 +159,35 @@ var (
 			TypedConfig: protoconv.MessageToAny(&statefulsession.StatefulSession{}),
 		},
 	}
+	InferencePoolExtProc = &hcm.HttpFilter{
+		Name: wellknown.HTTPExternalProcessing,
+		ConfigType: &hcm.HttpFilter_TypedConfig{
+			TypedConfig: protoconv.MessageToAny(&extproc.ExternalProcessor{
+				GrpcService: &core.GrpcService{
+					TargetSpecifier: &core.GrpcService_EnvoyGrpc_{
+						EnvoyGrpc: &core.GrpcService_EnvoyGrpc{
+							ClusterName: "dummy",
+						},
+					},
+					Timeout: &durationpb.Duration{Seconds: 10},
+				},
+				FailureModeAllow: true,
```

Comment:
TODO: Add support for [failureMode](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/v0.2.0/api/v1alpha2/inferencepool_types.go#L129-L134).

Reply:
Right, we should think about this a little more, theoretically, if I dont need faliureMode, original_dst is probably the easiest. I think. But TBD